pages/in/tx/history.vue

```diff
@@ -7,8 +7,8 @@
 
           <transaction-history
             ref="txHistory"
-            :user="getUserInfo.user"
-            :address="getUserHasWallet"
+            :user="getUserId"
+            :address="getUserERC20WalletAddress"
```

Comment:
The transaction API returns tx of cosmos too, so `getUserWalletAddress || getUserERC20WalletAddress`?
But this UI seems broken anyway

Reply:
Or is this erc20 guard intentional?

Reply:
User without ERC20 address will redirect to /in, I don't think users will come to this page anymore, there is no entry at all in like.co